code/game/sound.dm

```diff
@@ -88,6 +88,12 @@
 			var/mob/M = m
 			M.playsound_local(M, null, volume, vary, frequency, falloff, channel, pressure_affected, S)
 
+/proc/direct_sound_to_playing_players(sound_datum) //This is for sending specialized sound datums that playsound() can't account for, i.e. loops
```

Comment:
sound_to_playing_players now properly accounts for sound datums, so remove this.